### PR TITLE
[Appconfig] Move samples out of tests

### PR DESCRIPTION
**Pull request by maririos**
Following guidelines from: https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/repo-structure.md

### CODE REVIEW

**Review by pakrym — APPROVED** [review]

**Comment by weshaggard** [comment]
@KrzysztofCwalina for samples is the idea that they should be Tests or small applications? I think applications would be more interesting but I'm not sure what the arch board was hoping from for them.

**Comment by pakrym** [comment]
> I think applications would be more interesting but I'm not sure what the arch board was hoping from for them.

One advantage of them being tests is that we are sure they work without having to invent an additional infrastructure.

**Comment by weshaggard** [comment]
That is true but they are less useful for folks that want to try and write little apps so get started. As for verifying they don't break we should definitely come up with a strategy for that also, currently these are the first samples so I don't even thing we are building or running them yet in any case.

File: sdk/appconfiguration/Azure.ApplicationModel.Configuration/Azure.ApplicationModel.Configuration.sln [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.28803.308
+VisualStudioVersion = 16.0.28902.138
```
Comment (bsiegel) [inline sdk/appconfiguration/Azure.ApplicationModel.Configuration/Azure.ApplicationModel.Configuration.sln]: (This looks like a duplicate line btw)

Comment (maririos) [reply sdk/appconfiguration/Azure.ApplicationModel.Configuration/Azure.ApplicationModel.Configuration.sln]: Thanks! fixed

**Comment by maririos** [comment]
Agree that we need to decide between having samples as tests or as apps.
For now I am going to merge the PR and work on release AppConfig. Issue https://github.com/Azure/azure-sdk-for-net/issues/6150 created to follow up on the conversation

**Pull request merged**